tests/integ/modin/frame/test_aggregate.py

```diff
@@ -158,7 +158,14 @@ def test_corr_negative(numeric_native_df, method):
 @sql_count_checker(query_count=1)
 def test_string_sum(data, numeric_only_kwargs):
     eval_snowpark_pandas_result(
-        *create_test_dfs(data), lambda df: df.sum(**numeric_only_kwargs)
+        *create_test_dfs(data),
+        lambda df: df.sum(**numeric_only_kwargs),
+        # apparently, pandas doesn't propagate attrs if the frame is empty
```

Comment:
I'm assuming that we do propagate `attrs` in this case. If so, can we mention that in the comment?

Reply:
There are many similar occurrences in the other tests. Can we mention our behavior there as well?